onnx/defs/operator_sets.h

```diff
@@ -1185,6 +1185,16 @@ class OpSet_Onnx_ver21 {
   }
 };
 
+
+// Forward declarations for ai.onnx version 22
+
+// Iterate over schema from ai.onnx version 22
+class OpSet_Onnx_ver22 {
+ public:
+  static void ForEachSchema(std::function<void(OpSchema&&)> fn) {
```

Comment:
Curious what this does?

Reply:
I'm not sure. I was following https://github.com/onnx/onnx/blob/main/docs/OnnxReleases.md#after-pypi-release "Bump opset version for ai.onnx" section.

I realized that those steps could be done after the release branch is cut instead of waiting for the release to go public. However it looks like it causes the Linux CIs report a failure when it detects noting yet uses the `fn` inside of the method.

@liqunfu is this something that should still be done preemptively or should this wait until there's actually a change related to the next opset?

Reply:
> Curious what this does?

I can provide the history context here: without this previously ONNX developers might overlook creating new opset version here when adding new operators and there were a lot of back and forth communication for developers to change their PRs. Thus we think we can let the release manager always create the new opset here right after the release and in the future other developers won't need to worry about it.